src/static/js/modules/loan-comparison/mortgage-calculations.js

```diff
@@ -56,12 +64,12 @@ mortgage['monthly-hoa-dues'] = function (loan) {
 };
 
 mortgage['monthly-principal-interest'] = function (loan) {
-    return amortize({
-      amount: positive(loan['loan-amount']),
-      rate: loan['interest-rate'],
-      totalTerm: loan['loan-term'] * 12,
-      amortizeTerm: 60 // @todo loan term * 12?
-    }).payment;
+    return Math.round(amortize({
+                  amount: positive(loan['loan-amount']),
+                  rate: loan['interest-rate'],
+                  totalTerm: loan['loan-term'] * 12,
+                  amortizeTerm: 60 // @todo loan term * 12?
```

Comment:
I put this todo here ages ago because I wasn't sure how what the value of amortizeTerm should be - @amymok if you know the answer please share and we can probably remove this comment :)